index.html

```diff
@@ -0,0 +1,104 @@
+<!doctype html>
+<html class="no-js" lang="">
+  <head>
+    <meta charset="utf-8">
+    <meta http-equiv="X-UA-Compatible" content="IE=edge">
+    <title> Delicious | THE BEST FOOD BLOG ON THE WEB </title>
+    <meta name="description" content="">
```

Comment:
Consider putting something small in here, even just copy paste your title into content.